shared/js/ui/templates/shared/rating-hero.es6.js

```diff
@@ -76,3 +82,21 @@ function siteRatingSubtitle (isCalculating, rating, isWhitelisted) {
 
     return bel`${msg}`
 }
+
+// to avoid duplicating messages between the icon and the subtitle,
+// we combine information for both here
+function subtitleLabel (isCalculating, rating, isWhitelisted) {
+    if (isCalculating) return
+
+    if (isWhitelisted && rating.before) {
+        return `Privacy Protection Disabled, Privacy Grade ${rating.before}`
+    }
+
+    if (rating.before && rating.before === rating.after) {
+        return `Privacy Grade ${rating.before}`
+    }
+
+    if (rating.before && rating.after) {
+        return `Enhanced from ${rating.before} to ${rating.after}`
+    }
+}
```

Comment:
Fixes https://github.com/duckduckgo/duckduckgo-privacy-extension/issues/254 and https://github.com/duckduckgo/duckduckgo-privacy-extension/issues/253.

Reply:
BTW when we merge this, it invalidates my earlier comment in the new icons PR: https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/274#discussion_r208196682